proto/engine/v1/json_marshal_unmarshal.go

```diff
@@ -71,6 +91,26 @@ func (e *ExecutionBlock) UnmarshalJSON(enc []byte) error {
 	if !ok {
 		return errors.New("expected `totalDifficulty` field in JSON response")
 	}
+
+	rawWithdrawals, ok := decoded["withdrawals"]
+	if !ok || rawWithdrawals == nil {
+		e.Version = version.Bellatrix
```

Comment:
Do we know this for sure ? This just means there are 0 withdrawals in the payload.

Reply:
I asked on Discord and we have this now: https://github.com/ethereum/execution-apis/pull/334.